app/src/sharedTest/java/org/oppia/app/recyclerview/BindableAdapterTest.kt

```diff
@@ -381,28 +399,20 @@ class BindableAdapterTest {
     textView.text = "Value: " + data.intValue
   }
 
-  private fun getRecyclerViewListLiveData(
-    activity: BindableAdapterTestActivity
-  ): MutableLiveData<List<TestModel>> {
+  private fun getRecyclerViewListLiveData(activity: BindableAdapterTestActivity): MutableLiveData<List<TestModel>> { // ktlint-disable max-line-length
```

Comment:
Why not keep this method wrapped? We should only disable the line length check in exceptional cases (e.g. when we can't wrap/shorten something), and in this case it seems like we can properly avoid the situation just by wrapping the method declarations.

Reply:
Yes, I am thinking to fix such issues in files related to this pr in the last commit before meger, after final approval.

Reply:
Resolving this now on behalf of @BenHenning